Initial switch example using button and SVG images for graphical state

jongund opened the pull request.

jongund commented:
@mcking65 
Made update to move on/off labels from CSS to content, may need `aria-hidden`.

jesdaigle commented on examples/switch/switch-button.html:
```diff
@@ -0,0 +1,239 @@
+            </td>
+          </tr>
+          <tr data-test-id="switch-aria-checked">
+            <td></td>
```
why is this here?

mcking65 replied:
@jesdaigle  what are you referring to when you ask about "this"? Asking about the presence of the first cell in the row? The first ceel is for role, which is blank in this row b/c this is a row describing attributes of the row for the switch role.

a11ydoer commented:
[- check with NVDA - works well
- check with high contrast mode - dark and white theme
- The border disappers in dark theme(screen shot is attached)]

jongund commented:
@a11ydoer @mcking65 
* High contrast support has been fixed.
* Added `aria-hidden` to on and off labels
* Updated documentation to follow switch chackbox example

a11ydoer commented:
The [same comment](https://github.com/w3c/aria-practices/pull/1892#issuecomment-929427902) as in PR 1892

Switch can be turned on and off with double tap in an Android device. Need to Zoom in to interact with example is a separate responsive design issue.

mcking65 reviewed (approved):
Editorial review is complete.

I will add a link from the main doc and update the similar examples list after #1895  is merged and before merging this.

jongund commented:
@mcking65 
I made some updated to the accessibility feature documentation, so should be good to go now.
